### DISCUSSION TITLE
Get filterSelectOptions values from result of fetch or state?

### DISCUSSION THREAD

**Discussion by jeloooooo** [KevinVandy/material-react-table#355] [Q&A]
Is it possible to get the multi-select filterSelectOptions from the result of a fetch request? I'm trying to dynamically build the filterSelectOptions from the fetch values of a certain column. I placed the column values in an array state and set that array as my filterSelectOptions. However, when I click the multi-select filter, there are no options displayed but when I check the state, I can see that the state array is not empty and have the column values.

**Comment by KevinVandy** [comment]
What do your column definitions look like? Sometimes devs are forgetting to add those values to the `useMemo` dependency array.
